### PR TITLE
Jmm/aws s3 alternative

### PR DESCRIPTION
**Pull request by JeanMarie-TTS**
## PR checklist: submitters

- [ ]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.
- [ ]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [ ]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.
- [ ]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker-clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.
        
The larger the PR, the stricter we should be about these points.

### CODE REVIEW

**Comment by JeanMarie-TTS** [comment]
@asteel-gsa This is a PR to merge my changes into your branch.
Created fac_s3 , a django command that we can use instead of aws s3. I cloned backup_database.sh and created backup_local to test backups locally. backup_local.sh makes use of the fac_s3 tool to store log files ihe S3 bucket. You can take a look and make the necessary changes in backup_database.sh and in restore_database.sh.

File: backend/backup_local.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+BUCKET=gsa-fac-private-s3
+ENVIRONMENT=LOCAL
+#echo "Environment set as: $1"
+#export PATH=/home/vcap/deps/0/apt/usr/lib/postgresql/15/bin:$PATH
+DATE=$(date '+%Y-%m-%d-%H%M')
+LOG_FILE="$DATE"_backup_log.txt 
+DB_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-db-backup.psql.bin
+MEDIA_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-media-backup.tar
```
Comment (asteel-gsa) [inline backend/backup_local.sh:9]:
```suggestion
DB_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-db-backup.psql.bin
MEDIA_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-media-backup.tar
```
These should be removed, as they will be passed in from GHA

File: backend/backup_local.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+echo "DB Backup File: $DB_BACKUP_FILE" >> $LOG_FILE
+python manage.py dbbackup -o $DB_BACKUP_FILE >&1 >> $LOG_FILE
+# need to upload as my config backs up to local backup folder
+python manage.py fac_s3 $BUCKET --upload --src backup/$DB_BACKUP_FILE --tgt $DB_BACKUP_FILE
+
+echo "Media Backup File: $MEDIA_BACKUP_FILE" >> $LOG_FILE
+python manage.py mediabackup -o $MEDIA_BACKUP_FILE >&1 >> $LOG_FILE
+# need to upload as my config backs up to local backup folder
+python manage.py fac_s3 $BUCKET --upload --src backup/$MEDIA_BACKUP_FILE --tgt $MEDIA_BACKUP_FILE
```
Comment (asteel-gsa) [inline backend/backup_local.sh:21]:
```suggestion
python manage.py fac_s3 $BUCKET --upload --src backup/$MEDIA_BACKUP_FILE --tgt $MEDIA_BACKUP_FILE
```
$MEDIA_BACKUP_FILE should be $2

File: backend/backup_local.sh (hunk 1 above)
Comment (JeanMarie-TTS) [reply backend/backup_local.sh:9]: This file is only for testing locally in a docker env. I think we'll remove it from our codebase once we've made similar changes to backup_database.sh and tested it in the `dev` space.

**Comment by asteel-gsa** [comment]
@JeanMarie-TTS this is picking up my changes as well. Please pull down my branch and rewrite the bash script in `backup_database.sh`

**Pull request merged**